### DISCUSSION TITLE
TransferData(): Error in Embeddings(reduction)[nn.cells2, dims] : subscript out of bounds

### DISCUSSION THREAD

**Discussion by tsoruu** [satijalab/seurat#7860] [Q&A]
Hi, good community of Seurat, I have an error I don't know how to solve...
The error involves the TransferData() function. I've shared the screenshot of the code below.
![image](https://github.com/satijalab/seurat/assets/83799574/38676500-5d44-4732-b830-cfccee556ef8)

And this is the output of the code.
![image](https://github.com/satijalab/seurat/assets/83799574/0bf751ba-4265-43be-9234-e1058daa7f08)

I've also read a prior discussion regarding this issue (#3547), it mentions that this error could happen if too few anchor is found, but it is said that it found 108 anchor. Is it still too few of an anchor to be used or am I chasing the wrong conclusion for this error?
Help would be appreciated. Thanks in advance for reading, helping or even trying to help me...

**Comment by JerryZhang-1222** [comment]
Hi tsoruu, I have met the similar question as yours.

Please check that if you have subsetted your Signac object after finding anchors. If so, you may need to run `FindTransferAnchors` again. 

Best,
Feiyang.
